lib/cisco_node_utils/node.rb

```diff
@@ -419,8 +419,7 @@ def find_ascii(body, regex_query, *parent_cfg)
     else
       # find matches and return as array of String if it only does one
       # match in the regex. Otherwise return array of array
-      match = body.split("\n").map { |s| s.scan(regex_query) }
-      match = match.flatten(1)
+      match = body.scan(regex_query)
```

Comment:
By default Ruby regexps do not cross newlines (`.` does not match `\n`). Splitting the body into individual lines prevented explicitly multi-line regexps (as needed for XR in inventory.yaml) from working. This change permits those to work and has no impact to existing non-multiline regexps.